src/meinberlin/meinberlin/static/js/Packages/Meinberlin/Kiezkasse/Process/Module.ts

```diff
@@ -15,11 +10,7 @@ export var register = (angular) => {
     angular
         .module(moduleName, [
             AdhHttpModule.moduleName,
-            AdhMeinberlinPhaseModule.moduleName,
-            AdhMovingColumnsModule.moduleName,
-            AdhPermissionsModule.moduleName,
-            AdhTabsModule.moduleName,
-            AdhTopLevelStateModule.moduleName
+            AdhPermissionsModule.moduleName
```

Comment:
On the other hand, it is actually not necessary to explicitly depend on an angular module if one of our directives requires a directive from that module. So we can skip this dependency.